spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -69,51 +69,36 @@ private bool InitializeInstance()
         }
     }
 
-    private async Task DeleteStaleBackupsAndDatabasesAsync()
+
+    /// <summary>
+    /// Deletes first 5 database if no of database is more than 94.
+    /// </summary>
+    private async Task DeleteStaleDatabasesAsync()
     {
         DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
         var instanceName = InstanceName.FromProjectInstance(ProjectId, InstanceId);
         var databases = databaseAdminClient.ListDatabases(instanceName)
             .Where(c => c.DatabaseName.DatabaseId.StartsWith("my-db-") || c.DatabaseName.DatabaseId.StartsWith("my-restore-db-"));
```

Comment:
I think you might not want to filter here, just in case database other than "my-db-" and "my-restore-db-" are being created.

Reply:
We have one database named `my-test-database` for backup so keeping this.

Reply:
You don't need to, because when you try to convert the name to a long, it will fail, you will order by long.MaxValue and it will be the last DB in the list. Unless we delete all DBs, which won't happen, this one won't ever be deleted.

Reply:
Ah yes.